### DISCUSSION TITLE
DsHidMini Driver v2.0.248

### DISCUSSION THREAD

**Discussion by nefarius** [nefarius/DsHidMini#112] [Announcements]
# v2.0.248 DsHidMini changelog

This project was only possible with the investment of many **years** of learning, research and dedication. You're receiving this work for free, if it brought you joy, consider giving back and spend a few coins ❤️

[![DonateEUR](https://user-images.githubusercontent.com/286631/126007336-0b0cfa6d-b0bd-408f-9c1f-46c4227778fb.gif)](https://vigem.org/Donations/)

## How to install/update/uninstall

[Follow the installation guide](https://vigem.org/projects/DsHidMini/How-to-Install/).

## How to use in emulators and games

**You** need to configure the driver first for best experience!

Carefully read the provided documentation to configure the driver for best game compatibility. 

99% of your questions are already answered 😉 

👉 [About DsHidMini](https://vigem.org/projects/DsHidMini/)
👉 [How to install DsHidMini](https://vigem.org/projects/DsHidMini/How-to-Install/)
👉 [Info regarding using your DS3 as a Xbox/DS4 controller with DsHidMini](https://github.com/ViGEm/DsHidMini/issues/40)
👉 [DsHidMini's DS4 Mode user guide](https://vigem.org/projects/DsHidMini/DS4-Mode-User-Guide/)

## Remarks

🔥 **This release includes native XInput support** 🔥

⚠️ **This is a new major release which breaks backwards compatibility!** ⚠️

This release **is not compatible with** [BthPS3](https://github.com/ViGEm/BthPS3) version **1**.x.x! An **upgrade** to the latest **2**.x.x release is mandatory!

## Bugfixes

- Fixed rumble motor translation in XI mode (#111)

<hr /><em>This discussion was created from the release <a href='https://github.com/ViGEm/DsHidMini/releases/tag/v2.0.248.0'>DsHidMini Driver v2.0.248</a>.</em>

**Comment by TSnatch72** [comment]
do you need to fully remove 1.x.x versions of both BthPS3 and DsHidMini with all their drivers in order to upgrade to 2.x.x?

**Reply by Kanuan** [reply]
- You won't be able to install BthPS3 v2 if the version 1 is already installed
- It should be possible to just install DsHidMini v2 over the first version. If you run into problems fully remove all versions and try again
- Also, don't forget to install the new driver by also installing the `igfilter.inf` file. It's required in v2 alongside the dshidmini.inf

If you run into trouble, remember to refer to the [install guide](https://vigem.org/projects/DsHidMini/How-to-Install/#removal)

**Comment by pupphelper** [comment]


**Reply by nefarius** [reply]
Make a new discussion and don't hijack one that is not related to your topic, thank you ❤️

**Reply by pupphelper** [reply]
> Make a new discussion and don't hijack one that is not related to your topic, thank you ❤️

Very sorry.  created a new discussion.  Thanks for all you do for the community. its truly appreciated!

**Comment by jemspena453-cpu** [comment]
como conecto mi pando de ps3 ah mi lapto
